app/main/views/services.py

```diff
@@ -8,9 +8,11 @@
 from app.main import helpers
 from ... import db
 from ...models import ArchivedService, Service, Supplier, Framework
-from ...validation import validate_json_or_400, \
-    validate_updater_json_or_400, is_valid_service_id
-from ..utils import url_for, pagination_links, drop_foreign_fields, link
+import traceback
```

Comment:
Yes - leftover from the merge hell!  I removed it at least three times during rebasing, but it's somehow got back in. Will sort it.